### PR TITLE
CCM-954

### PR DESCRIPTION
**Pull request by PhilipSkinner**
## Summary
Adds documentation onto the POST /v1/message-batches endpoint.

Adds documentation should a 201 response be generated.

## Reviews Required
* [ ] Dev
* [ ] Test
* [ ] Tech Author
* [x] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/communications-manager.yaml [hunk 1]
```diff
@@ -49,6 +49,25 @@ paths:
...
         Use this endpoint to send a batch of messages to 1 or more NHS patients.
 
+        ### References
+
+        You must provide two reference values within the payload to this endpoint:
+
+        * A message batch reference
+        * A per message reference
+
+        Both of these references must be UUIDs.
```
Comment (LEUNHU) [inline specification/communications-manager.yaml:59]: Given UUID is an acronym, should we use the full term followed by (UUIDs)

**Review by LEUNHU — CHANGES_REQUESTED** [review]
1 comment on use of acronyms

File: specification/communications-manager.yaml (hunk 1 above)
Comment (PhilipSkinner) [reply specification/communications-manager.yaml:59]: Good shout, I'll make this part better.

Comment (PhilipSkinner) [reply specification/communications-manager.yaml]: I've given the full term, the acronym plus I've linked to wikipedia so the consumer can find more information on UUIDs if they have never used them before.

**Review by LEUNHU — APPROVED** [review]

**Pull request merged**